openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -1252,11 +1252,11 @@
     "aggregate-domain-type-description": "Domains closer to online services and transactional databases that include events, and transactional data.",
     "airflow-guide-message": "OpenMetadata uses Airflow to run Ingestion Connectors. We developed Managed APIs to deploy ingestion connectors. Please use OpenMetadata Airflow instance or refer to the guide below to install the managed APIs in your Airflow installation.",
     "airflow-host-ip-address": "OpenMetadata will connect to your resource from the IP {{hostIp}}. Make sure to allow inbound traffic in your network security settings.",
-    "alerts-action-description": "Select important events like 'Schema Change' or 'Test Failure' to generate an alert.",
     "alerts-description": "Stay current with timely alerts using webhooks.",
     "alerts-destination-description": "Send notifications to Slack, MS Teams, Email, or use Webhooks.",
     "alerts-filter-description": "Specify the change events to narrow the scope of your alerts.",
-    "alerts-trigger-description": "Trigger for all data assets or a specific entity.",
+    "alerts-source-description": "Specify the source that will activate the alert.",
+    "alerts-trigger-description": "Select important trigger events like 'Schema Change' or 'Test Failure' to generate an alert.",
```

Comment:
Please validate these changes, en-us has 2 changes, and other language only has single  key change